src/restful/controllers/ratingCategoryController.js

```diff
@@ -0,0 +1,124 @@
+import Response from "../../system/helpers/Response";
+import RatingCategoryService from "../../services/ratingCategoryService";
+import * as v from "valibot";
+import { ratingCategorySchema } from "../../system/validators";
+import { filterValidationError } from "../../system/utils";
+
+const {
+  createRatingCategory,
+  findAllRatingCategories,
+  findRatingCategoryById,
+  checkRatingCategoryExists,
+  updateRatingCategoryName,
+} = RatingCategoryService;
+
+export default class RatingCategoryController {
+  static async createRatingCategory(req, res) {
+    try {
+      const results = v.safeParse(ratingCategorySchema, req.body);

```

Comment:
@TresorRw it could be better if we have some validation middleware which accept the schema and return the result instead of importing this "valibot" in every files.

Reply:
Thanks @leandreAlly, Let me look into that

Reply:
Hello @leandreAlly, Kindly review again and let me know